x/evm/keeper/msg_server.go

```diff
@@ -1463,12 +1463,17 @@ func (s msgServer) SignCommands(c context.Context, req *types.SignCommandsReques
 
 	commandList := types.CommandIDsToStrings(commandBatch.GetCommandIDs())
 	for _, commandID := range commandList {
-		s.Logger(ctx).Info(fmt.Sprintf("signing command %s in batch %s for chain %s using key %s", commandID, batchedCommandsIDHex, chain.Name, string(commandBatch.GetKeyID())))
+		s.Logger(ctx).Info(fmt.Sprintf("signing command %s in batch %s for chain %s using key %s", commandID, batchedCommandsIDHex, chain.Name, string(commandBatch.GetKeyID())),
+			types.AttributeKeyChain, chain.Name,
+			tsstypes.AttributeKeyKeyID, string(commandBatch.GetKeyID()),
+			"commandBatchID", batchedCommandsIDHex,
+			"commandID", commandID,
```

Comment:
@milapsheth

Reply:
Can we please keep the values in the log message for now? We should refactor logs at the same time for consistency, otherwise it's gonna be harder during debugging.

Reply:
Reverted to the original log message with the attributes included as well